crates/net/network/src/network.rs

```diff
@@ -235,7 +235,7 @@ impl NetworkInfo for NetworkHandle {
         let status = self.get_status().await?;
 
         Ok(NetworkStatus {
-            client_name: "Reth".to_string(),
+            client_version: "Reth".to_string(),
```

Comment:
this should actually be included in the `fn get_status`

do you want to modify the `fn get_status` in a followup, so that it returns `NetworkStatus` instead?

Reply:
Can you elaborate? Not sure to get what is needed

Reply:
`Network:: get_status`

https://github.com/paradigmxyz/reth/blob/def59c399658caee5a0e2124d9b11cf23f711a97/crates/net/network/src/network.rs#L132-L137

should return `NetworkStatus` directly